### DISCUSSION TITLE
关于机器人无法收到某一类型的事件（如入群邀请）的讨论

### DISCUSSION THREAD

**Discussion by hqshi** [Mrs4s/go-cqhttp#636] [General]
**环境信息**
go-cqhttp版本: go-cqhttp-v0.9.40-windows-amd64
运行环境: win10
连接方式: 反向websocket
使用协议: ipad

本人在使用go-cqhttp机器人遇上了无法收到通过搜索群号加群的事件的问题。依据教程排除了如下问题
1. 机器人版本更新不及时
2. 机器人没有审批权限
3. 机器人被风控

同时发现
在开启DEBUG模式查看日志，发现未接收到相关事件，同时其他事件均可以正常接收。手机、电脑客户端都可以收到入群邀请消息

根据用户交流群里的大佬提示，发现是由于除了ipad协议的机器人以外还挂了手机qq和电脑qq的客户端所导致无法收到事件。
在重新测试后发现电脑QQ客户端会导致ipad协议的机器人无法收到加群申请，同时手机QQ则不会遇到这个问题

这里分享我遇到的问题当抛砖引玉，也希望讨论区能有更多的问题解决方法分享

**Comment by Mrs4s** [comment]
应该是相关事件解析错误或忽略导致的, 节后我测试下
